app/views/entitlements/_entitlements_table.html.erb

```diff
@@ -18,20 +18,22 @@
 
 <table class="entitlements-table">
   <tbody>
-    <tr>
-      <td>
-        <h3>Total Weight of Household Goods</h3>
+    <%- if @entitlement_search.nil? or @entitlement_search.dependency_status == 'false' -%>
```

Comment:
I think these conditionals are simple enough now. I couldn't figure out a good place to put a helper so I decided to make entitlements.total_weight_self_plus_dependents a required field. That seemed appropriate because prior to that one rank did not have that field. All the other ranks had that field, although for many of them it was the same as total_weight.